inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -843,7 +843,7 @@ impl_types layout_optimizer::get_preferred_impl_type(program_node& node) {
         const size_t kKeyValue = kBatchNum * std::min(kClassNum, static_cast<size_t>(8)) * kNStreams;
         preferred_impl = (kKeyValue > 64) ? impl_types::ocl : impl_types::cpu;
     } else if (node.is_type<reorder>()) {
-        if (!node.get_program().get_engine().get_device_info().supports_immad)
+        if (!_optimization_attributes.use_onednn_impls)
```

Comment:
now it is merged. Please see: https://github.com/openvinotoolkit/openvino/pull/7716